corehq/apps/accounting/templates/accounting/email/customer_subscription_ending_reminder.html

```diff
@@ -4,17 +4,14 @@
 <p>
     {% blocktrans %}
     Your {{ plan_name }} subscription ends soon! If you do not renew your subscription before then,
-    you will automatically be subscribed to the CommCare Community plan {{ ending_on }}
-    and lose access to all pay-only features on the following domains: {{ domains }}.
+    you will automatically be subscribed to the CommCare Community plan and
+    lose access to all pay-only features {{ ending_on }}
     {% endblocktrans %}
 </p>
 
 <p>
     {% blocktrans %}
-    To change or renew your subscription, please log into your project on
-    CommCare HQ and navigate to the
-    <a href="{{ subscription_url }}">subscription page</a> in your
-    project settings.
+    To change or renew your subscription, please reach out to <a href="mailto:accounts@dimagi.com">sales@dimagi.com</a>.
```

Comment:
The email addresses should be passed in as context variables listed in settings.py:
https://github.com/dimagi/commcare-hq/blob/68826225ad6afd306db83e50b642e65ae7ea0f28/settings.py#L470

with the values set here:
https://github.com/dimagi/commcare-cloud/blob/e1e90baf40b54f40f5493890d920881bcd55da03/src/commcare_cloud/ansible/group_vars/all.yml#L101